friendly-captcha/public/mutation-observer.js

```diff
@@ -0,0 +1,44 @@
+(function () {
```

Comment:
Looks great - the only thing to change is to add support for IE11:

You will have to change the `for... of` loops to be plain loops (or use `.forEach`).


If you want to support even older browsers, you would replace all `let` and `const` with `var` - but I think those browsers don't even account for `0.05%` of all browsers, and WP doesn't support them anyhow in all likelihood.

Reply:
Good point! 👍